src/app/components/autocomplete/autocomplete.component.ts

```diff
@@ -57,35 +74,48 @@ export class AutocompleteComponent implements OnInit, OnDestroy, ControlValueAcc
 	};
 
 	ngOnInit(): void {
-		this.addPackage = new FormControl('', {
+		this.addOption = new FormControl('', {
 			nonNullable: true,
 			asyncValidators: this.errorHandlerService.noDuplicatesValidator(this.loadedEntities$),
 		});
-		this.addPackage.valueChanges.pipe(takeUntil(this.unsubscribe$)).subscribe((value) => this.onChange(value));
+		this.addOption.valueChanges.pipe(takeUntil(this.unsubscribe$)).subscribe((value) => this.onChange(value));
 	}
 
 	ngOnDestroy(): void {
 		this.unsubscribe$.next();
 		this.unsubscribe$.complete();
 	}
 
+	/**
+	 * On enter click, select first element from the autocomplete dropdown if exists
+	 */
+	onChooseFirstOption() {
```

Comment:
minor nitpick- if the method name is going to begin with 'on', I feel like it should be in the form of 'onSomeEvent'. (So 'onEventKey' or something in this case) I'm not sure that's the best choice though, and simply calling it 'chooseFirstOption' would probably make more sense.

Reply:
Yeah, renaming the method makes sense. I renamed it to onEnterKey, check [ee64ccd](https://github.com/bitovi/opensource-analytics-dashboard/pull/18/commits/ee64ccd341b635b405d90118341bff692645e557) .

I personally like to use the prefix `on` when that method is called from the HTML. But I am not against not using it, depends on how we decide. I am open to both versions.

Reply:
I like this change, `onEnterKey` sounds right to me